### DISCUSSION TITLE
AudioBookShelf appreciation thread

### DISCUSSION THREAD

**Discussion by ZLoth** [advplyr/audiobookshelf#2964] [General]
Let me take a few minutes to express my appreciation to the programmer of AudioBookShelf. I've been looking for a way to stream the audiobooks that I had purchased from Humble Bundle, plus transfer my Audible collection to my own server. I kept hoping Plex would add audiobook functionality, but it's been more than a decade since the feature was first requested, and we're still waiting. I was trying out Emby, but the functionality and the details of AudioBookShelf blows Emby out of the water. I know I keep suggesting improvements to the software to make it easier to use, but at the core, it's a very excellent player, with features I wish Audible had. 

Thank you for the hard work in making this project a success.

**Comment by GiveMeDaTaco** [comment]
Almost a year later, and this post is still just as relevant. Just want to add my huge "thank you" to the pile of gratitude. Really, REALLY appreciate all the work done on this project.
